### DISCUSSION TITLE
Can this be used together with Landamic

### DISCUSSION THREAD

**Discussion by GreXXL** [lucky-media/resume#1] [Q&A]
I was wondering if this could be used in combination with landmaic - and if yes how so :)

Thanks!

**Comment by GreXXL** [comment]
As this is the first discussion, I don't know if it gets seen. Just to make sure a ping @lokmanm or @arlindmusliu 

Thanks!

**Reply by lokmanm** [reply]
Hi @GreXXL,

`Technically` its possible but you will have to do some manual work to get both configs to work with each other.

You will have to edit the `webpack.mix.js` so the output is different for both of them. You will have `resume.css` for the files in the resume. 

Let me know if this helps. Here is also an [article](https://ralphjsmit.com/tailwind-css-multiple-configurations-laravel-mix) for this.
